gnome/gingerblue/Portfile

```diff
@@ -10,8 +10,8 @@ platforms           darwin
 license             GPL-3+
 maintainers         {gnome.org:ole @oleaamot} \
                     openmaintainer
-description         GNOME Gingerblue
-long_description    Locate Internet Radio Stations
+description         GNOME Gingerblue is Free Music Software for GTK+/GNOME.
+long_description    It supports immediate audio recording in compressed Ogg encoded audio files stored in the ~/Music/ directory with XSPF 1.0 playlist from the line input on a computer or remote audio cards through USB connection through PipeWire with GStreamer.
```

Comment:
Could this be wrapped to 80 columns?

Reply:
The long_description can't be wrapped.

Reply:
Every other port does it. Look at how another port handles this.

Reply:
Fixed in the latest commit.